src/cephadm/cephadmlib/daemons/smb.py

```diff
@@ -58,6 +58,8 @@ class Config:
     smb_port: int
     ceph_config_entity: str
     vhostname: str
+    smbmetrics_image: str
+    smbmetrics_port: int
```

Comment:
This is nitpicky, but at this context saying *smb*metrics seems redundant. How about just `metrics_image` and `metrics_port`? If not, please `smb_metrics_x` for consistency with other var names.

Reply:
Fair enough -- removed the `smb` prefix.